AppCenterAnalytics/AppCenterAnalytics/MSAnalytics.m

```diff
@@ -98,30 +99,34 @@ - (NSString *)groupId {
 - (void)applyEnabledState:(BOOL)isEnabled {
   [super applyEnabledState:isEnabled];
   if (isEnabled) {
+    if (self.startedFromApplication) {
 
-    // Start session tracker.
-    [self.sessionTracker start];
+      // Start session tracker.
+      [self.sessionTracker start];
```

Comment:
Shouldn't we do all of this also when the configuration is updated with an app secret?

Reply:
`applyEnabledState` is called from update callback.